workflowtrigger/util/event/kafka_events.go

```diff
@@ -2,6 +2,7 @@ package event
 
```

Comment:
For all the serialization in these methods, order does not matter. Is it possible to automatically insert all the keys using the OrderedMap (not add them explicitly)?

Reply:
Updated those methods with `map[string]interface{}` return value.